src/client/Microsoft.Identity.Client/Platforms/Features/DefaultOSBrowser/HttpListenerInterceptor.cs

```diff
@@ -89,6 +89,13 @@ public HttpListenerInterceptor(ICoreLogger logger)
                 cancellationToken.ThrowIfCancellationRequested();
 
                 // if cancellation was not requested, propagate original ex
+                if (ex is HttpListenerException)
+                {
+                    throw new Exception("HttpListenerException occurred when listening for the response from the browser. " + 
```

Comment:
>Exception [](start = 30, length = 9)

nit: should this be an `MsalClientException`?

Reply:
Yes, `MsalClientException`. There should even be an analyzer that prevents you from raising simple Exception, as it is too "general".

Reply:
Done.